hi.txt

```diff
@@ -0,0 +1,2 @@
+hi world
+hello
```

Comment:
please remove this and swap with `hello world`

Reply:
I have fixed it. please review again @omawang

Reply:
ok thanks